daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -66,7 +66,7 @@ private[speedy] sealed abstract class SBuiltin(val arity: Int) {
     */
   private[speedy] def execute(args: util.ArrayList[SValue], machine: Machine): Control
 
-  protected def unexpectedType(i: Int, expected: String, found: SValue) =
```

Comment:
Majority of changes here are Boy Scout work (see end of this file for more meaningful changes)